pom.xml

```diff
@@ -59,6 +59,12 @@
             <optional>true</optional>
         </dependency>
 
+        <dependency>
+            <groupId>software.amazon.awssdk</groupId>
+            <artifactId>kms</artifactId>
+            <version>2.20.91</version>
```

Comment:
You do not need to declare a version here.
Read https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/get-started.html
In particular, expand the POM.
```suggestion
```